specs/commandline/sign.md

```diff
@@ -27,58 +41,78 @@ Flags:
 
 ```shell
-# Prerequisites: 
-# - A compliant signing plugin is installed in notation. See notation plugin documentation (https://github.com/notaryproject/notaryproject/blob/main/specs/plugin-extensibility.md) for more details.
-# - User creates keys and certificates in a 3rd party key provider (e.g. key vault, key management service). The signing plugin installed in previous step must support generating signatures using this key provider.
+$ notation sign localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+Successfully signed localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9.
+```
 
-# Add a default signing key referencing the key identifier for the remote key, and the plugin associated with it.
-notation key add --default --name <key_name> --plugin <plugin_name> --id <remote_key_id>
+### Sign an OCI artifact using COSE signature format
 
-# sign a container image using a remote key
-notation sign <registry>/<repository>:<tag>
+```shell
+# Prerequisites: 
+# A default signing key is configured using CLI "notation key"
+
+# Use option "--signature-format" to set the signature format to COSE.
+$ notation sign --signature-format cose <registry>/<repository>@<digest>
 ```
 
-### Sign an OCI artifact using the default signing key
+### Sign an OCI artifact stored in a registry using the default signing key
 
 ```shell
 # Prerequisites: 
 # A default signing key is configured using CLI "notation key"
 
 # Use a digest that uniquely and immutably identifies an OCI artifact.
-notation sign <registry>/<repository>@<digest>
+$ notation sign <registry>/<repository>@<digest>
 ```
 
-### Sign a container image and specify the signature expiry duration, for example 1 day
+### Sign an OCI artifact stored in a registry and specify the signature expiry duration, for example 24 hours
 
 ```shell
-notation sign --expiry 1d <registry>/<repository>:<tag>
+notation sign --expiry 24h <registry>/<repository>@<digest>
 ```
 
-### Sign a container image using a specified signing key
+### Sign an OCI artifact stored in a registry using a specified signing key
 
 ```shell
 # List signing keys to get the key name
-notation key list
+$ notation key list
 
 # Sign a container image using the specified key name
-notation sign --key <key_name> <registry>/<repository>:<tag>
+$ notation sign --key <key_name> <registry>/<repository>@<digest>
+```
+
+### Sign an OCI artifact identified by a tag
+
+```shell
+# Prerequisites: 
+# A default signing key is configured using CLI "notation key"
+
+# Use a tag to identify a container image
+$ notation sign <registry>/<repository>:<tag>
+```
+
+An example for a successful signing:
+
+```shell
+$ notation sign localhost:5000/net-monitor:v1
+Warning: Always sign the artifact using digest(`@sha256:...`) rather than a tag(`:latest`) because tags are mutable and a tag reference can point to a different artifact than the one signed.
```

Comment:
```suggestion
Warning: Always sign the artifact using digest(`@sha256:...`) rather than a tag(`:v1`) because tags are mutable and a tag reference can point to a different artifact than the one signed.
```